ruby/ql/test/library-tests/ast/erb/Erb.expected

```diff
@@ -50,7 +50,6 @@ erbAstNodes
 | template.html.erb:31:5:31:9 |  end  |
 | template.html.erb:31:10:31:11 | %> |
 | template.html.erb:31:12:33:5 | \n</ul>\n<%%>\n |
-| template.html.erb:33:1:33:3 | <%% |
```

Comment:
This token does not seem to overlap with any other one, so we probably shouldn't drop it.